src/components/spreadsheet/utils/config-tables.js

```diff
@@ -3298,6 +3298,58 @@ export const TABLES_DEFINITIONS = {
             },
         ],
     },
+    BUSES: {
+        index: 14,
+        name: 'Bus',
+        type: EQUIPMENT_TYPES.BUSBAR_SECTION,
```

Comment:
BUS instead of BUSBAR_SECTION
see remarks in  PR : https://github.com/gridsuite/network-map-server/pull/160